modules/tinymce/src/core/main/ts/newline/InsertLi.ts

```diff
@@ -13,6 +13,10 @@ const hasFirstChild = (elm, name) => {
   return elm.firstChild && elm.firstChild.nodeName === name;
 };
 
+const isFirstChild = (elm: HTMLElement) => {
+  return elm.parentNode.firstChild === elm;
```

Comment:
This may not be safe, as `parentNode` is potentially `null` (though where it's called it should never be null). Maybe do this instead to be safe?
```
return elm.parentNode?.firstChild === elm;
```